package.json

```diff
@@ -23,6 +23,7 @@
     "iconv"                 : "~2.3.0",
     "ipaddr.js"             : "~1.5.0",
     "js-yaml"               : "~3.9.0",
+    "logfmt"                : "^1.2.0",
```

Comment:
Shouldn't this be an optional dependency?

Reply:
Nope, this is being used for default too, line 275 of logger.js